### PR TITLE
Migrate Chicago to Open Civic Data models django-councilmatic.

### PR DESCRIPTION
**Pull request by fgregg**
This PR, in concert with https://github.com/datamade/django-councilmatic/pull/240, gets Chicago councilmatic up and going.

TODO:

- [x] unhack migrations
- [x] get tests working
- [x] get notifications working

### CODE REVIEW

**Comment by hancush** [comment]
## setup

### in django-councilmatic

1. clone, if needed: `https://github.com/datamade/django-councilmatic.git`
2. checkout the branch with changes: `git checkout ocd`

### in chi councilmatic

3. do a local install of django-councilmatic: `pip install -e /path/to/django-councilmatic`
4. create a database: `createdb chicago_councilmatic`
5. install postgis: `psql -d chicago_councilmatic -c "create extension postgis"`
6. run the migrations: `python manage.py migrate`

### in scrapers-us-municipal

8. clone, if needed: `https://github.com/opencivicdata/scrapers-us-municipal.git`
7. create a pupa settings file: `touch pupa_settings.py`
8. add pupa to installed apps
9. set `DATABASE_URL` to your chicago councilmatic db

    ```
    INSTALLED_APPS = (
        'django.contrib.contenttypes',
        'opencivicdata.core.apps.BaseConfig',
        'opencivicdata.legislative.apps.BaseConfig',
        'pupa',
        'councilmatic_core'
    )

    DATABASE_URL = 'postgres:///chicago_councilmatic'
    ```

10. initialize the database: `pupa dbinit us`
11. add some data: `pupa update chicago --fastmode --rpm=0`

File: councilmatic/settings_jurisdiction.py [hunk 1]
```diff
@@ -4,10 +4,10 @@
 # These settings are required #
 ###############################
 
-OCD_CITY_COUNCIL_ID = 'ocd-organization/ef168607-9135-4177-ad8e-c1f7a4806c3a'
-CITY_COUNCIL_NAME = 'Chicago City Council'
+OCD_CITY_COUNCIL_ID = 'ocd-organization/3266e69a-85ed-4c76-a98f-5652081ab5d1'
```
Comment (hancush) [inline councilmatic/settings_jurisdiction.py]: i'm still not sure why this change was necessary?

Comment (hancush) [reply councilmatic/settings_jurisdiction.py]: i bet it's because it's referring to my local org, not the one in the api. oop.

Comment (hancush) [reply councilmatic/settings_jurisdiction.py]: the id is created when the organization is created, i.e., there isn't a canonical id to reference here, because it will be created when we initialize the chicago-specific ocd database. i've removed references to the id, and instead use the org name to filter, in https://github.com/datamade/chi-councilmatic/pull/252/commits/c35675acbe4976546485dccf111bd9c0b1b7a3e5 and upstream in https://github.com/datamade/django-councilmatic/pull/240/commits/373a77bf086aba7cb2057901a7980482935b6fdc.

**Comment by hancush** [comment]
This PR updates `chi-councilmatic` to cooperate with the changes described in https://github.com/datamade/django-councilmatic/pull/240#issuecomment-503221198. It should reproduce all functionality on the existing Chicago Councilmatic site, with the exception of importing shapes and notification support.

File: chicago/views.py [hunk 2]
```diff
@@ -1,32 +1,34 @@
...
 class ChicagoIndexView(IndexView):
     template_name = 'chicago/index.html'
     bill_model = ChicagoBill
     event_model = ChicagoEvent
 
     def last_meeting(self):
-        return ChicagoEvent.objects.filter(name__exact=settings.CITY_COUNCIL_MEETING_NAME)\
-                           .filter(start_time__lt=datetime.now())\
-                           .latest('start_time')
+        return ChicagoEvent.objects.annotate(start_date_dt=Cast('start_date', DateTimeField())).filter(name__exact=settings.CITY_COUNCIL_MEETING_NAME)\
```
Comment (fgregg) [inline chicago/views.py]: it would be better to move this annotation into django councilmatic, if not possible to move into Chicago model.

Comment (hancush) [reply chicago/views.py]: done! one already existed upstream, so i used that instead.

**Comment by fgregg** [comment]
- [ ] Primary sponsored legislation on council member pages seem to show all sponsorships not just primary sponsorships. https://chicago-staging.councilmatic.org/person/sigcho-lopez-byron-1060341871ce/

**Comment by fgregg** [comment]
- [ ] pin requirements

**Comment by fgregg** [comment]
@hancush do you want to do a final review before merging into master?
